src/main/java/org/zalando/nakadi/repository/kafka/KafkaFactory.java

```diff
@@ -137,13 +139,23 @@ public void terminateProducer(final Producer<String, String> producer) {
         }
     }
 
+    public Consumer<String, String> getConsumer(final Properties properties) {
```

Comment:
can we maybe inline this method? Because it is used only in another getConsumer method.